rhobs/README.md

```diff
@@ -0,0 +1,18 @@
+# RHTAP Alerting Rules
+RHTAP alerting rules are deployed to RHOBS (Observatorium) using app-interface.
+The rules are defined inside the [alerting](alerting) directory and are referenced using
+the combination of the repository URL, the rules path and the git branch or commit hash.
+
+In the case of RHTAP, we prefer the commit hash over the branch name as it provides
+better control over changes to the rules being deployed.
+
+This means that whenever the rules here have changed, we a change in app-interface is
```

Comment:
I think you added that "we" as a mistake in this line

Reply:
done